SPECS/intel-idv-services/intel-idv-services.spec

```
@@ -0,0 +1,90 @@
+%define systemd_user_dir %{buildroot}/usr/lib/systemd/user
+%define systemd_system_dir %{buildroot}/etc/systemd/system
+%define local_bin_dir %{buildroot}/usr/local/bin
+
+Name:           intel-idv-services
+Version:        0.1
+Release:        1%{?dist}
+Summary:        A package to install scripts and systemd services for Intelligent Desktop Virtualization
+Distribution:   Edge Microvisor Toolkit
+Vendor:         Intel Corporation
+License:        Apache-2.0
+URL:            https://github.com/open-edge-platform/edge-desktop-virtualization
+Source0:        https://github.com/open-edge-platform/edge-desktop-virtualization/releases/download/pre-release-v%{version}/%{name}-%{version}.tar.gz
+Source1:        90-default.preset
+
+BuildArch:      noarch
+Requires(post): systemd
+Requires(preun): systemd
+
+%description
+This package installs the scripts folder to /usr/local/bin/idv, enables and starts a root-level systemd service, and enables and starts a user-level systemd service.
+
+%prep
+%setup -q
+
+%build
+
+%install
+# Copy the scripts folder to /usr/local/bin/idv
+mkdir -p %{local_bin_dir}/idv
+cp -r init /%{local_bin_dir}/idv
```

/ before %{local_bin_dir} appears to be a typo, should be just %{local_bin_dir}/idv.
```suggestion
cp -r init %{local_bin_dir}/idv
```

---

This has been taken care in the commit - [09aa130](https://github.com/open-edge-platform/edge-microvisor-toolkit/pull/214/commits/09aa130afb8e18cd57e749225ceea5f1fce4bd9f)